code/game/gamemodes/objectives_subtypes.dm

```diff
@@ -0,0 +1,58 @@
+/datum/objective/debrain/command
+	name = "Debrain command"
+	target_jobs = list("Captain", "Head of Personnel", "Head of Security", "Research Director", "Chief Engineer", "Chief Medical Officer", "Quartermaster")
```

Comment:
This would be much more sane if this used a departmental strings or departmental flags, and then used a global proc to get job list from global lists like `command_head_positions`